src/cdk/overlay/position/connected-position-strategy.ts

```diff
@@ -265,6 +265,15 @@ export class ConnectedPositionStrategy implements PositionStrategy {
     return this;
   }
 
+  /**
+   * Sets the origin element, relative to which to position the overlay.
+   * @param origin Reference to the new origin element.
+   */
+  withOrigin(origin: ElementRef): this {
```

Comment:
`setOrigin` for this, since it's not something you'd specify when building the strategy?

Reply:
Done